modules/ROOT/pages/deployment-groups.adoc

```diff
@@ -0,0 +1,10 @@
+= About Deployment Groups (Early Access)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Deployment Groups is a Cloudhub Entity that allows you to combine multiple regions together. After a Deployment Group is created multiple applications can be deployed to this deployment group. Deployment Group allows a single application to be deployed in an active-active state to multiple workers in multiple regions simultaneously.
```

Comment:
If you have global deployment entitlement enabled, you can use a deployment group to combine multiple workers in multiple regions into one CloudHub entity and deploy a single application to them all. The deployment group can be shared among environments in a business group.